src/cryptoadvance/specter/templates/includes/tx-table.html

```diff
@@ -942,10 +942,14 @@ <h1>{{ _("Export transactions to CSV") }}</h1>
                 // If there are multiple pages show the pagination bar
                 if (this.pageCount > 1) {
                     this.paginationContainer.classList.remove('hidden');
+                    this.paginationCounter.classList.remove('hidden');
                     this.paginationCounter.innerText = `of ${this.pageCount}`;
                     this.paginationIdxInput.max = this.pageCount;
                 } else {
+                    console.log("we are running in pageCount is less or equal to 1")
```

Comment:
This can be commented.
Also, shouldn't we hide the entire pagination counter when it is just a single page?

Reply:
That is just from debugging, will delete it.
Was thinking about hidding the pagination completely, but it somehow looked smoother to keep Page 1 - but I don't have strong opinion here.